server/src/main/java/io/druid/segment/realtime/plumber/Plumber.java

```diff
@@ -44,6 +47,7 @@
    *
    * @param commitRunnable code to run after persisting data
    */
+  void persist(Committer commitRunnable);
   void persist(Runnable commitRunnable);
```

Comment:
Do we still need persist(Runnable)? People could submit a Committer with null metadata, and that would simplify Plumbers.